R/FromNCDFSG.R

```diff
@@ -0,0 +1,121 @@
+#'@title Convert NetCDF to sp objects
+#'
+#'
+#'@param nc_file A string file path to the nc file to be read.
+#'
+#'@description
+#'Attemps to convert a NetCDF-CF DSG Simple Geometry file into an sp object.
+#'
+#'@references
+#'https://github.com/bekozi/netCDF-CF-simple-geometry
+#'
+#'@importFrom ncdf4 nc_open nc_close ncvar_get ncatt_get
+#'@importFrom sp Polygon Polygons SpatialPolygons SpatialPolygonsDataFrame CRS Line Lines SpatialLines SpatialLinesDataFrame SpatialPointsDataFrame
+#'
+#'@export
+FromNCDFSG = function(nc_file) {
+
+  nc <- nc_open(nc_file)
+
+  checkVals <- checkNCDF(nc)
+
+  instance_id<-checkVals$instance_id
+  instanceDim<-checkVals$instanceDim
```

Comment:
Agreed... I don't think one has been decided on, but I've been following tidyverse elsewhere. So underscores. I'll do some find and replace action.